tests/containers/podman_pods.pm

```diff
@@ -40,6 +40,41 @@ sub run {
         record_info('Cleanup', 'Stop pods');
         assert_script_run('podman play kube --down hello-kubic.yaml');
     }
+
+    unless (is_sle('<=15-SP2')) {
+        # Kube generate
+        record_info('Test', 'Generate the yaml from a pod');
+        assert_script_run('podman pod create testing_pod');
+        image = "registry.suse.com/bci/bci-busybox:latest";
+        script_retry("podman pull -q $image", timeout => 300, delay => 60, retry => 3);
+        assert_script_run("podman container create --pod testing_pod --name container image sh -c \"while true; do sleep 3600; done\"");
```

Comment:
```suggestion
        assert_script_run("podman container create --pod testing_pod --name container $image sleep 3600");
```
You forgot the `$` :)

Also, I think we can drop the `while` loop and just use `sleep 3600` as we don't expect the container to run that longer.